core/src/test/java/org/pentaho/platform/genericfile/BaseGenericFileProviderTest.java

```diff
@@ -60,11 +65,18 @@ public BaseGenericFileTree getTreeCore( @NonNull GetTreeOptions options ) {
       throw new UnsupportedOperationException();
     }
 
+
     @Override
     public boolean hasAccess( @NonNull GenericFilePath path, EnumSet<GenericFilePermission> permissions ) {
       throw new UnsupportedOperationException();
     }
 
+    //TODO implement
+    @Override public IGenericFileContentWrapper getFileContentWrapper( @NonNull String path )
+      throws FileNotFoundException, OperationFailedException, FileSystemException {
+      return null;
```

Comment:
```suggestion
      return null;
```

if this is only an EE feature then you should throw new` java.lang.UnsupportedOperationException` or something similar.

Reply:
I will follow the pattern as the other methods in this class are doing of `throw new UnsupportedOperationException();`